NitroxServer/Serialization/SaveDataUpgrades/Upgrade_V1800.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+using NitroxModel.DataStructures;
+using NitroxServer.Serialization.Upgrade;
+
+namespace NitroxServer.Serialization.SaveDataUpgrades
+{
+    public sealed class Upgrade_V1800 : SaveDataUpgrade
+    {
+        public override Version TargetVersion { get; } = new Version(1, 8, 0, 0);
+
+        protected override void UpgradeWorldData(JObject data)
+        {
+            data["InventoryData"]["EquippedItems"] = new JObject(new Dictionary<string, NitroxId>());
```

Comment:
Is this actually upgrading? Do we want to if we can move old to new to the other? It might not be possible with these changes but we should try if we can

Reply:
Bc of the massive changes to the base piece data we unfortunately can not provide upgradability from V1.7.